.github/workflows/testing_multiversion.yml

```diff
@@ -1,17 +1,15 @@
-name: unit tests [Python 3.8|3.9|3.10]
+name: unit tests [Python 3.8|3.9|3.10|3.11]
 
 on:
   push:
-    branches: [ main ]
-    paths-ignore:
```

Comment:
Any reason this path was removed? It would end up causing a recursive action run.

Reply:
see above